### ISSUE TITLE
Make invalid Multisignature address

### ISSUE THREAD

**Issue by liray-unendlich** [phoreproject/openbazaar-go#16]
Brief Description: Openbazaar-go makes invalid escrow address

Operating System (OS and version): Windows8.1
Phore Marketplace version (shown on `About Phore Marketplace` page in menu): v1.3.3
Hardware: 

Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce): Always

Steps to reproduce:
1. Choose item
2. Click "BUY NOW" button
3. Choose escrow payment
4. Check escrow address

Observed Behavior:
Openbazaar-go makes invalid escrow address(bc1qc~).

Expected Behavior:
Valid escrow address should start from 6~

Additional info (links, images, etc go here): 
![image](https://user-images.githubusercontent.com/15893314/38079537-1ecf7078-337a-11e8-9d0e-720cefbaad50.png)
This is log for openbazaar-go
[LOG.zip](https://github.com/phoreproject/openbazaar-go/files/1859303/LOG.zip)

**Comment by liray-unendlich** [comment]
Thanks for fixing about multisignature address.
I tried escrow payment.
I found there is something wrong about escrow
Sellar can confirm escrow payment, and buyer can fullfilment and sellar can complete.
But after order completion, escrow service doesn't send charge.
I think this problem come from transaction fee and count confirmation transaction(count confirmation, not on time)
After completion, sellar and buyer can't check their order like below picture.
(I restart openbazaard two times)
![image](https://user-images.githubusercontent.com/15893314/38776625-ebb5f582-40d4-11e8-83b7-6a88571961ea.png)

Here is buyer and seller ob.log
buyer: 
[buyer.log](https://github.com/phoreproject/openbazaar-go/files/1911980/buyer.log)

seller:
[seller.log](https://github.com/phoreproject/openbazaar-go/files/1911981/seller.log)
